db/models.py

```diff
@@ -1,4 +1,9 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import UniqueConstraint
+
+import settings
```

Comment:
it's better to use
`from django.conf import settings` because it defines main settings for sure.
It's possible that you have multiple settings files or a folder named "settings", thus python will incorrectly import settings